tests/test_database.py

```diff
@@ -4,31 +4,17 @@
 import database_connector
 import database_manager
 
-
-def create_test_database(database_connector):
-    database_connector.execute("CREATE DATABASE test_db")
-
-
-def set_test_database(database_connector):
-    database_connector.set_database("test_db")
-
-def set_database_to_none(database_connector):
-    database_connector.set_database(None)
-
-
-def delete_test_database(database_connector):
-    database_connector.execute("DROP DATABASE test_db")
-
-
-def create_items_test_table(database_connector):
+def create_test_database_setup(database_connector):
+    database_connector.execute("CREATE DATABASE TestDB") #create database
```

Comment:
Prefer comments on the line above, and always have space after \#
If you keep the # here, ppl usually have two spaces before the \#

Same for other comments